src/Sarif/CodeGenHints.json

```diff
@@ -1,12 +1,64 @@
 {
+  "annotatedCodeLocation": [
+    {
+      "kind": "BaseTypeHint",
+      "arguments": {
+        "baseTypeNames": [
+          "PropertyBagHolder"
+        ]
+      }
+    }
+  ],
   "AnnotatedCodeLocation.Properties": [
     {
-      "kind": "DictionaryHint"
+      "kind": "DictionaryHint",
+      "arguments": {
+        "valueTypeName": "SerializedPropertyInfo",
+        "namespaceName": "Microsoft.CodeAnalysis.Sarif.Readers",
```

Comment:
What's the namespace hint for?

---
In reply to: [62749197](https://github.com/Microsoft/sarif-sdk/pull/257#discussion_r62749197) [](ancestors = 62749197)